wallet/commands.go

```diff
@@ -343,6 +343,18 @@ func (wal *Wallet) IsAddressValid(address string) (bool, error) {
 	return wall.IsAddressValid(address), nil
 }
 
+// VerifyNessage checks if the given message match with the signature for the specified wallet address.
```

Comment:
"Verify**M**essage checks if the given message match**es** the signature for the address."